### DISCUSSION TITLE
Location of items on Map

### DISCUSSION THREAD

**Discussion by offron** [piotrulos/MSCModLoader#146] [Help me] [answered]
Thanks for all the hard work!
I have a quick question. 
When loosing modded items on the map, is there a way to find out its position without reseting the mod?
The MSC Editor wont find them.

**Comment by piotrulos** [comment, marked as answer]
There is no standard provided for modded save files, every mod uses different save method.

It's up to individual mods how they handle lost items. 

MSC Editor will never find them, because none of the mods (as far as I know) use the vanilla msc save system

**Reply by offron** [reply]
Is there a chance to implement a guideline? 

So everything could be unified

I mean you are the main reason we have simple mod dev/installation, so it seem reasonable.

Would simplify for the future alot. Could also benefit msc editor

**Reply by piotrulos** [reply]
Implementing isn't an issue, biggest issue is making everyone follow that guideline.

Also some modders use custom encrypted save files, that can't be edited at all.
